libs/cardano-ledger-binary/src/Cardano/Ledger/Binary/Group.hs

```diff
@@ -26,10 +27,11 @@ instance (DecCBORGroup a, EncCBORGroup a) => DecCBOR (CBORGroup a) where
   decCBOR = CBORGroup <$> groupRecord
 
 instance EncCBORGroup a => EncCBOR (CBORGroup a) where
-  encCBOR (CBORGroup x) = encodeListLen (listLen x) <> encCBORGroup x
+  encCBOR (CBORGroup x) = encodeListLen (listLen $ Proxy @a) <> encCBORGroup x
 
 groupRecord :: forall a s. (EncCBORGroup a, DecCBORGroup a) => Decoder s a
-groupRecord = decodeRecordNamed "CBORGroup" (fromIntegral . toInteger . listLen) decCBORGroup
+groupRecord =
+  decodeRecordNamed "CBORGroup" (fromIntegral . toInteger . const (listLen $ Proxy @a)) decCBORGroup
```

Comment:
Looks like I missed one more suggestion. Using `Proxy @a` would have been fine, it is the `fromIntegral . toInteger` that is totally unnecessary here, which was there before this PR.
```suggestion
  decodeRecordNamed "CBORGroup" (listLenInt . (`asProxyTypeOf` Proxy)) decCBORGroup
```